src/AppInstallerRepositoryCore/CompositeSource.cpp

```diff
@@ -327,163 +340,127 @@ namespace AppInstaller::Repository
             std::shared_ptr<IPackageVersion> m_trackingPackageVersion;
         };
 
-        // A composite package for the CompositeSource.
-        struct CompositePackage : public IPackage
+        // A composite package for the installed package of a CompositePackage.

```

Comment:
"A composite package for [...] a CompositePackage"? Should it just be "A package for [...]"?